packages/itinerary-body/src/otp-react-redux/view-stop-button.tsx

```diff
@@ -1,18 +1,19 @@
 import React, { Component, ReactElement } from "react";
+import { Stop } from "@opentripplanner/types";
 import { FormattedMessage } from "react-intl";
 
 import * as S from "../styled";
 import { defaultMessages } from "../util";
 
 interface Props {
-  onStopClick: ({ stopId: string }) => void;
-  stopId: string;
+  onStopClick: (stop: Stop) => void;
```

Comment:
This is a breaking change. Either:
1. Add breaking change commits to all packages that use this new method signature, or,
2. If maintaining compatibility, change the signature of `onStopClick` to `(stop: Stop | { stopId: string })` (You could create an alias `type StopEventHanlder = (stop: Stop | { stopId: string }) => void` in the types package to avoid repetition.)

Reply:
That's a great idea! I'll add it. Thank you for the suggestion